doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_pm_optimization.rst

```diff
@@ -0,0 +1,214 @@
+
+Latency in local domains
+========================
+
+Any local software module (like a device driver) can anticipate events like ISRs.
+Some of these events have predictable timing, while others have unpredictable timing.
+Handling the latency of events with unpredictable timing is the same in both simple and complex systems.
+
+If handling an event with predictable timing requires restoring the state of the software module or the peripherals used by this module before the event is processed, the software module is responsible for scheduling a timer event in advance.
+This scheduled event is used to restore the state of the software module or peripherals.
+
+The Power Management subsystem in a local domain is responsible for scheduling a wake-up in advance to compensate for the domain's core state restoration latency from the local power state.
+The wake-up time scheduled in advance by the power management subsystem is combined with the advance time added by the software module.
+This approach ensures that the local domain and the software modules anticipating an event have sufficient time to fully restore before the event occurs, allowing the event to be handled without latency.
+
+Unretained hardware classification
+**********************************
+
+Some power states in the nRF54H20 SoC result in powering off certain peripherals.
+The state of these peripherals is not retained by hardware and must be restored by software before the peripheral is activated.
+
+See the following sections for the lists of peripheral groups and the related software modules responsible for restoring the peripheral's state for each group.
+
+Peripherals in local domains
+============================
+
+All local domains include a common set of hardware modules.
+In addition to these, most local domains also contain domain-specific peripherals.
+
+Common peripherals for all local domains
+----------------------------------------
+
+Each local domain contains a set of peripherals that are classified consistently across all local domains.
+The following table summarizes the active peripherals that need handling when exiting the *Local Suspend to RAM* state.
+
++---------------+--------------------+--------------------+--------------------+--------------------------+
+|Type           | List of the        | Source of data to  | Time of restoration| Software module          |
+|               | peripherals        | restore            |                    | responsible for restoring|
++===============+====================+====================+====================+==========================+
+|Active         | * ``MVDMA``        | Device driver's    | Decided by the     | The device driver        |
+|peripherals    |                    | code and data      | driver             |                          |
```

Comment:
I don't think we have the MVDMA drive available atm, so maybe it's better to remove it, or to mark that this is a design, not the implementation state.

Reply:
Ok I'm removing the entire section then